src/runtime_src/core/tools/xbmgmt2/XBUtilities.cpp

```diff
@@ -0,0 +1,162 @@
+/**

```

Comment:
Is this file specific to xbmgmt or is it a verbatim copy from xbutil?
It latter then lets refactor to share common code.

Reply:
I was planning to refactor this code in the next PR under a new dir tools/common